Article/Article.js

```diff
@@ -3,18 +3,26 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.domElement = domElement;
+    this.closeButton = domElement.querySelector(".closeButton");
+    // create a reference to the ".expandButton" class.
+    this.expandButton = domElement.querySelector(".expandButton");
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.buttons = domElement.querySelector(".buttons");
+    // this.closeButton.textContent = "close";
+    this.expandButton.textContent = "expand";
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener("click", () => {
+      this.expandArticle();
+    });
   }
 
   expandArticle() {
     // Using our reference to the domElement, toggle a class to expand or hide the article.
-
+    this.domElement.classList.toggle("article-open");
+    this.expandButton.classList.toggle("hide-btn");
+    this.closeButton.classList.toggle("hide-btn");
+    console.log(this.closeButton);
```

Comment:
ah! i see now. instead of hiding the button and replacing it, it's easier to use an if statement to change the text of the button in reaction to which it already has when this function is invoked.